runtime/runtime-params-estimator/src/testbed.rs

```diff
@@ -15,8 +15,6 @@ use std::sync::Arc;
 
 pub struct RuntimeTestbed {
     /// Directory where we temporarily keep the storage.
-    #[allow(dead_code)]
-    pub(crate) workdir: tempfile::TempDir,
```

Comment:
This needs to be here to make sure it's not dropped prematurely (dropping `TempDir` value deletes the underlying temporary directory). I suggest the following change insteed:

```
_workdir: tempfile::TempDir,
```

* remove `pub`
* use `_` rather than allow to silence the warning. The `_` prefix is idiomatic way to signal "this is here only for drops".

Reply:
Changed.
Thanks for the review.